.vscode/settings.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
I don't think we should check this in.

Reply:
Removed the whole folder, not sure how it slipped through my initial check